spyder/plugins/editor/utils/editor.py

```diff
@@ -51,6 +51,23 @@ def drift_color(base_color, factor=110):
             return base_color.lighter(factor + 10)
 
 
+def block_safe(block):
```

Comment:
Let's rename this function to `is_block_safe` for clarity.

Reply:
done